### PR TITLE
standardize time and dates

### LINKED ISSUE #217: Standardize time/date
All time/date should be stored and compared in UTC for attendance. It should be presented in the UI in the user's timezone.

This needs to be cleaned up a lot

### PR DESCRIPTION
**Pull request by jackpalaia**
closes #217

### CODE REVIEW

File: src/screens/Events/Admin/Statistics/EditEventStatsForm.jsx [hunk 1]
```diff
@@ -33,16 +34,32 @@ const Styled = {
   `,
 };
 
+const timeValidator = object().shape({
```
Comment (colin4554) [inline src/screens/Events/Admin/Statistics/EditEventStatsForm.jsx]: Can this be moved to the eventHelpers file?

**Review by colin4554 — APPROVED** [review]

**Comment by YiQin93** [comment]
There is a small issue with editing the check-out time on the time cards:
- Edit volunteer check-in and out time doesn't update until refreshing the page - if I previously typed in an incorrect check-out time (a check-out time that is earlier than the check-in time) during this update
- Please find more details in this report: https://go.playerzero.app/share/63f427f2e059aae6d7a6ecb2

But this would be a low priority as this event has a small probability.

File: src/screens/Events/Admin/Statistics/EditEventStatsForm.jsx (hunk 1 above)
Comment (jackpalaia) [reply src/screens/Events/Admin/Statistics/EditEventStatsForm.jsx]: done

**Comment by YiQin93** [comment]
This works perfectly now!!

**Pull request merged**